content/Video_Display.md

```diff
@@ -609,12 +609,12 @@ four bytes with the following meanings:
 
 ### Byte0 - Y Position
 
-Specifies the sprites vertical position on the screen (minus 16). An
+Specifies the sprites vertical position on the screen (plus 16). An
```

Comment:
Since both ways around make sense in their own way ("write the position plus 16" / "the value there is the position when 16 is subtracted"), I'd rather write e.g. "Specifies the sprite's vertical position on the screen, with an offset of 16 pixels. For example, writing 17 will have the sprite's topmost row on the third screen line."

While we're at this, please correct "sprites" to "sprite**'**s".

Finally, we may want to further clarify the behavior for partially off-screen values (e.g. 2, or 154; careful with 8x16 mode)